extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -80,10 +86,10 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.type_ = null;
 
     /**
-     * @private {Object<string, string>} A map of request names to the request
-     * format string used by the tag to send data
+     * @private {Object<string, !RequestData>} Maps request names to configs.
```

Comment:
Yes, or more generally by supporting POST. We need to separate out the path from the rest of the payload. Since the payload may not be a url in the case of a POST, there is no systematic way to separate path from payload.